app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,395 @@
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/{portalId}/teams/{teamId}/developers \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --header 'accept: */*' \
+  --data '{"id":"DEVELOPER-ID"}'
+```
+  If you don't remember the IDs for your test developers, you can sent a [GET request to list developers](https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Developers/list-portal-developers).<br><br>
+  You should get a `201` response like the following:
+  ```bash
+  Created
+  ```
+<!-- 1. App registration step, I don't see any APIs that allow us to do this-->
+
+You now have added your test developer accounts to two teams with different access permissions for your Dev Portal. These developers won't be able to access anything in Dev Portal yet, though, because we need to enable Portal RBAC next.
+
+So far, you've used the following endpoints:
+* [Create team](https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Teams/create-portal-team)
+* [Assign role](https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Team%20Roles/portal-teams-assign-role)
+
+Review the API documentation to see what other features are available.
+{% endnavtab %}
+{% endnavtabs %}
+ 
+
+### Enable Portal RBAC
+
+Now that you've configured your two different teams and assigned permissions to those teams, you can enable Portal RBAC to let those permissions take affect. Portal RBAC is disabled by default in the {{site.konnect_short_name}} portal. 
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal** and click **Settings** in the sidebar.
+1. To turn on Portal RBAC, click the **Portal RBAC** toggle.
+
+Now, if you access your Dev Portal as one of the test developer accounts, you should see that the one assigned to the "Prospective Partners" team can only view the Pizza Ordering API while the one assigned to the "Authorized Delivery Partners" team can both view and consume the Pizza Ordering API.
+{% endnavtab %}
+{% navtab API %}
+To enable RBAC, you must make a `PATCH` request to the portal configuration endpoint. The following example shows how to enable RBAC in the portal. For more details on how to create your personal access token, see [Authentication](/konnect/api/#authentication). 
+
+```bash
+curl --request PATCH \
+  --url https://<region>.api.konghq.com/konnect-api/api/portals/{portal-id} \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{"rbac_enabled": true}'
+```
+
+The `portal-id` can be found in the {{site.konnect_short_name}} UI in the **Dev Portal** section. <br><br>
```

Comment:
Should we link the endpoint here so the reader can look deeper into it?

Reply:
Talked on Slack, but we can't do this right now because the Portal RBAC enable endpoint is v0 right now.